src/preview.js

```diff
@@ -1,51 +1,70 @@
-import countries from '../assets/countries.json';
+/**
+ * WordPress dependencies
+ */
+import { __, sprintf } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+ import countries from '../assets/countries.json';
 import continentNames from '../assets/continent-names.json';
 import continents from '../assets/continents.json';
 import { getEmojiFlag } from './utils';
-import { __, sprintf } from '@wordpress/i18n';
 
 export default function Preview( { countryCode, relatedPosts } ) {
 	if ( ! countryCode ) return null;
 
 	const emojiFlag = getEmojiFlag( countryCode ),
-	      hasRelatedPosts = relatedPosts?.length > 0;
+		hasRelatedPosts = relatedPosts?.length > 0;
 
 	return (
 		<div className="xwp-country-card">
```

Comment:
It'd be great to improve the card semantics. This could be a `section`. Thanks to that, the main heading could become `h1`, the related posts heading would be `h2` and the related post titles would remain as `h3`s. Also, the flag container – `.xwp-country-card__media` – could become a `figure`, and the related posts container – `.xwp-country-card__related-posts` – could be e.g. an `aside`.